install-php-locally.md

```diff
@@ -0,0 +1,101 @@
+# Installing PHP Locally (for Symfony CLI)
+
+The [Symfony CLI](https://symfony.com/doc/current/setup/symfony_cli.html) is awesome for developing Symfony
+applications locally! It enables you to run a local web server, integrates with Docker containers for
+services like a database, and provides many other features to streamline your development workflow.
+It's mostly batteries included, but it does require PHP to be installed locally.
+
+We use it here at SymfonyCasts for almost all our courses. Let's look at how to install PHP on different
+operating systems. We'll also make sure extensions for the most common databases (MySQL, SQLite, PostgreSQL)
+are installed.
+
+Make sure you have the [Symfony CLI installed](https://symfony.com/download) before getting started!
+
+## macOS (Homebrew)
+
+First, ensure you have [Homebrew](https://brew.sh) installed.
+
+Next, install PHP:
+
+```terminal
+brew install php
+```
+
+This will install the latest stable version. Ensure it's working by running:
+
+```terminal
+symfony check:requirements
+```
```

Comment:
I would move this to the end, out of the OS-related section. Because no matter how you install the PHP, this command will be the same for any OS.

Reply:
On the other hand, that's what you always do after installing something, check if it works

Reply:
I'm going to move to a final section (that all OS-specific sections will direct to)